gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/JavaServiceDistributionPlugin.groovy

```diff
@@ -77,15 +77,18 @@ class JavaServiceDistributionPlugin implements Plugin<Project> {
             checkScript.configure(distributionExtension.serviceName, distributionExtension.checkArgs)
         }
 
+        distributionExtension.productDependenciesConfig = project.configurations.getByName("runtime")
```

Comment:
if done when applying plugin, then the configuration must exist, meaning the java plugin must have already added it. if there is a way to delay this or evaluate it lazily, that would be ideal. maybe move to right above line 84, in the afterEvaluate?